schema/filter/DefaultFilterStrategy.go

```diff
@@ -28,34 +26,39 @@ import (
 type defaultFilterStrategy struct {
 	extractFields th2MsgFieldExtraction
 
-	Logger zerolog.Logger
+	logger zerolog.Logger
 }
 
-var Default filter.FilterStrategy = defaultFilterStrategy{Logger: zerolog.New(os.Stdout).With().Timestamp().Logger()}
+var Default FilterStrategy = defaultFilterStrategy{logger: zerolog.New(os.Stdout).With().Str("component", "default_filter_strategy").Timestamp().Logger()}
 
 func (dfs defaultFilterStrategy) Verify(messages *p_buff.MessageGroupBatch, filters []mqFilter.MqRouterFilterConfiguration) bool {
 	// returns true if MessageGroupBatch entirely matches at least one filter(any) from list of filters in the queueConfig,
 	// returns true if filters are not at all
 	// returns false otherwise
 	if len(filters) == 0 {
-		dfs.Logger.Debug().Msg("No filters for MessageGroupBatch")
 		return true
 	}
-	dfs.Logger.Debug().Msg("Started filtering of MessageGroupBatch for sending")
+	dfs.logger.Debug().Msg("Started filtering of MessageGroupBatch for sending")
 	for n, flt := range filters {
 		res := true
 		for _, msgGroup := range messages.Groups {
 			if !dfs.CheckValues(msgGroup, flt) {
 				res = false
-				if e := log.Debug(); e.Enabled() {
-					e.Msgf("MessageGroup with first message id: %v didn't match filter #%v : metadata - %v , message: %v ", ExtractID(msgGroup), n+1, flt.Metadata.Filters, flt.Message.Filters)
+				if e := dfs.logger.Debug(); e.Enabled() {
+					e.Int("filter N", n+1).
+						Interface("Metadata", flt.Metadata.Filters).
+						Msgf("Filter fields to which didn't match MessageGroupBatch with first message id: %v  ", IDFromMsgGroup(msgGroup))
```

Comment:
I think it is better to add `IDFromMsgGroup` as `Interface` to the log event instead of using it in the message